done 3-module

skifzp opened the pull request.

skifzp committed:
done 3-module/1-task

skifzp committed:
done 3-module/2-task

skifzp committed:
3-3

skifzp committed:
Merge branch 'master' of https://github.com/js-tasks-ru/jsbasic-20220418_skifzp
added

skifzp committed:
3-4

skifzp committed:
Merge branch 'master' of https://github.com/js-tasks-ru/jsbasic-20220418_skifzp
merge

skifzp committed:
done 3-module

jsru commented:
Проверьте, пожалуйста, ваше решение, не все тесты прошли (PR не будет принят до тех пор, пока **все** добавленные задачи не будут решены).

skifzp committed:
reformed code

jsru commented:
Добавляю преподавателя (@koalex) для код-ревью.

skifzp committed:
added 4-module
Merge branch 'master' of https://github.com/js-tasks-ru/jsbasic-20220418_skifzp

skifzp committed:
done 4-module/1-task makeFriendsList()

skifzp committed:
added 4-module/2-task
Merge branch 'master' of https://github.com/js-tasks-ru/jsbasic-20220418_skifzp

skifzp committed:
done 4-module/2-task makeDiagonalRed()

skifzp committed:
added 4-module/3-task
Merge branch 'master' of https://github.com/js-tasks-ru/jsbasic-20220418_skifzp

skifzp committed:
done 4-module/3-task highlight()

jsru commented:
Решение было обновлено, посмотрим что скажет @koalex
